config/clickhouse/users_config.xml

```diff
@@ -0,0 +1,121 @@
+<clickhouse>
+    <!-- See also the files in users.d directory where the settings can be overridden. -->
+
+    <!-- Profiles of settings. -->
+    <profiles>
+        <!-- Default settings. -->
+        <default>
+            <allow_experimental_analyzer>0</allow_experimental_analyzer>
```

Comment:
it seems like a lot of the failing tests could be because this setting is enabled by default in the newer versions. I think maybe we could be overwritting these settings in `users.d` directory but wasn't sure if that was for different profiles and I wanted to set this as the default